integration-cli/daemon/daemon.go

```diff
@@ -217,7 +217,7 @@ func (d *Daemon) StartWithLogFile(out *os.File, providedArgs ...string) error {
 		return errors.Wrapf(err, "[%s] could not find docker binary in $PATH", d.id)
 	}
 	args := append(d.GlobalFlags,
-		"--containerd", "/var/run/docker/libcontainerd/docker-containerd.sock",
+		"--containerd", "/var/run/balena/libcontainerd/containerd.sock",
```

Comment:
Can we not diverge from the convention of this socket file being in the format `<name>-containerd.sock`. Is there a reason to?

Reply:
@zozo123 did you mean `balena-containerd.sock` here?